webui/src/components/_commons/PanelServiceDetails.vue

```diff
@@ -54,7 +54,7 @@
         </div>
       </q-card-section>
 
-      <q-card-section v-if="data.loadBalancer.terminationDelay">
+      <q-card-section v-if="data.loadBalancer && data.loadBalancer.terminationDelay">
```

Comment:
data itself is never nil? i.e., is there no need for:

"data && data.loadBalancer && data.loadBalancer.terminationDelay" 

?